src/Microsoft.VisualStudio.LanguageServices.Razor/BackgroundDocumentGenerator.cs

```diff
@@ -257,23 +257,35 @@ private void ProjectManager_Changed(object sender, ProjectChangeEventArgs e)
                     }
 
                 case ProjectChangeKind.DocumentAdded:
+                case ProjectChangeKind.DocumentChanged:
                     {
-                        var project = _projectManager.GetLoadedProject(e.ProjectFilePath);
-                        Enqueue(project, project.GetDocument(e.DocumentFilePath));
+                        var project = e.Newer;
```

Comment:
`Older` and `Newer` were both never assigned. How are these not nullref'ing below? Or is there no test covering this code path? Or is Github lying to me?

Reply:
Yeah wierd.